lib/index.js

```diff
@@ -178,6 +178,34 @@ function _ensure_cheerio(value) {
     return value;
 }
 
+class DictionaryAsset extends BaseAsset {
+    constructor() {
+        super();
+        this._object_type = "DictionaryObject";
+        this._content_type = "text/html";
```

Comment:
I might suggest having some sort of rendered HTML in the document body anyway, even if it's not used directly in the discovery feed...